src/main/java/com/dragonballs/controllers/UserController.java

```diff
@@ -0,0 +1,73 @@
+package com.dragonballs.controllers;
+
+import com.dragonballs.entities.User;
+import com.dragonballs.services.authentication.JwtAuthenticationResponse;
+import com.dragonballs.services.authentication.JwtTokenUtil;
+import com.dragonballs.services.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.servlet.http.HttpServletRequest;
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class UserController {
+    private int AUTH_TOKEN_SUBSTRING_VALUE = 7;
+    @Autowired
+    private UserService userService;
+
+    @Autowired
+    private JwtTokenUtil jwtTokenUtil;
```

Comment:
As discussed with Evelina - RestController is just a bad place for jwtTokenUtil - I think it should be extracted to another class